Remove the type and vendor field in the toast notification of storage model

jdjgya opened the pull request:
### What type of PR is this?

Refactor

<br/>

### Which issue(s) this PR fixes?

None

<br/>

### What this PR does?

- Remove the type and vendor field in the toast notification of storage model

- Adding the STG00004I/E(verification) and STG00005I/E(setting to default) (both have been updated to the spreadsheet as well https://docs.google.com/spreadsheets/d/196Xy8s6K4PcvpJjQIxX4VAFQzI_ETY2DTbOAODT7AaY/edit?gid=1674446536#gid=1674446536)

- Replace the validating with verifying since designer changed the wording.

jdjgya committed:
fix(storage integration): update the incorrect model update api for external storage and remove the type and vendor field in the toast notification

raven-pan reviewed (commented):
Hi @jdjgya Thanks!

I found that the `type` and `vendor` fields are still in the `additionalInfo`.

<img width="514" height="953" alt="Image" src="https://github.com/user-attachments/assets/63649767-3f3e-4589-83de-836d771a063f" />

jdjgya committed:
fix(storage integration): remove the type and vendor in the model event and replace the validating with verifying

jdjgya committed:
fix(storage integration): add STG00004I/E(verification) and STG00005I/E(setting to default) for model storage

jdjgya commented:
Hi @raven-pan , Thanks a lot for reminding this to me. 

I've removed the `vendor` and `type` from the places above

<br/>

moreover, as discussed, I've also added another changes in this PR as well,

- adding the STG00004I/E(verification) and STG00005I/E(setting to default) (both have been updated to the spreadsheet as well https://docs.google.com/spreadsheets/d/196Xy8s6K4PcvpJjQIxX4VAFQzI_ETY2DTbOAODT7AaY/edit?gid=1674446536#gid=1674446536)

- replace the `validating` with `verifying` since designer changed the wording.


Please help check when you're available, Thanks a lot 🙏

raven-pan reviewed (approved):
LGTM, Thanks for your work!
Just a minor discussion.

raven-pan commented on docs.yaml:
```diff
@@ -12848,11 +12848,10 @@ components:
                     - creating
                     - updating
                     - deleting
-                    - setting to default
```
Hi @jdjgya

Maybe we should keep the `setting to default` enum.
